src/main/services/pdfGenerator.ts

```diff
@@ -0,0 +1,42 @@
+import * as express from 'express'
+
+import { Claim } from 'app/claims/models/claim'
+import { DocumentsClient } from 'app/documents/documentsClient'
+import * as http from 'http'
+import * as HttpStatus from 'http-status-codes'
+
+const documentsClient: DocumentsClient = new DocumentsClient()
+
+export class PdfGenerator {
+  static async requestHandler (req: express.Request, res: express.Response, next: express.NextFunction): Promise<void> {
+    {
+      const { externalId } = req.params
+      documentsClient.getClaimIssueReceiptPDF(externalId, res.locals.user.bearerToken)
```

Comment:
this handler is called `PdfGenerator#requestHandler` but it is hardcoded to download the claim issue receipt, the response receipt can also be downloaded too